src/templates/head-meta.js

```diff
@@ -0,0 +1,24 @@
+import { LOCALE_DIR_KEY, seo } from './options'
+
+import { nuxtI18nSeo } from './seo-head'
+
+export const headMeta = function () {
+  if (!this.$i18n) {
+    return {}
+  }
+
+  let metaObject
+  if (seo) {
+    metaObject = nuxtI18nSeo.call(this)
+  } else {
+    metaObject = { htmlAttrs: {} }
+  }
+
+  const currentLocale = this.$i18n.localeProperties || {}
+  const currentLocaleDir = currentLocale[LOCALE_DIR_KEY]
```

Comment:
when dropping it, the tests depends on `loader` fixture or `loader-yaml` will fail and it leads to crash `can't find dir property of undefined`, i investigated more and i've noticed something weird, `localeProperties` is not defined **ONLY** in the components where `<i18n> component` is used and the `vueI18nLoader` is set to true for sure. 

i don't know how/why this happening, but if it can be considered as a bug, It needs to be fixed in general, since `localeProperties` will be undefined when used in those components.

Reply:
That means that we're missing an assignment in `extendVueI18nInstance` that would copy that property on creating a new `vue-i18n` instance.

Reply:
copying? can u elaborate more? i will create a separate PR for this. 

i was investigating earlier today, and it worked when i assigned the property this way, but i think it's not the correct way, not sure. 

```JS
 const extendVueI18nInstance = i18n => {
    i18n.localeProperties = klona(locales.find(l => l[LOCALE_CODE_KEY] === i18n.locale) || { code: '' })
  }
```

Reply:
Yes, something like that looks good.

`vue-i18n` creates a local instance of itself when a local `<i18n>` component is used. It copies all properties it knows about by itself but the Nuxt-specific ones have to be copied (or re-created) from that `extendVueI18nInstance` listener.

Reply:
Awesome, thanks for the clarification.